boards/arm/atsame54_xpro/atsame54_xpro.yaml

```diff
@@ -12,6 +12,7 @@ toolchain:
   - xtools
 supported:
   - gpio
+  - pwm
```

Comment:
Not a problem with entry, but I2C is on wrong position.

Reply:
I don't think we currently have a repository-wide convention to keep this list alphabetically ordered; in particular, this file does not have an alphabetically ordered list, so I put the `pwm` entry where I thought it made the most sense.

Maybe we can do a mass clean-up for these later if deemed necessary.

Reply:
I was wondering if we can start to do it when we are touching that places, with time we get all organized. This way we lost less time, I think, because it will be like fix a typo.
Anyway, in this case since the list is already out of order, it is up to you.

Reply:
Hopefully such lists will go away and we'll get the data from devicetree.